### ISSUE TITLE
Lookupfallback of map causes error

### ISSUE THREAD

**Issue by DorienD** [zotonic/zotonic#2193]
We have found that this commit: https://github.com/zotonic/zotonic/commit/e8a443fc6d580cdf6edc51453fa6472f92d488b1

causes this error:

```
16:31:45.805 [error] erfgoedbrabant z_transport:201 Error 'undef' for transport message {z_msg_v1,1,false,<<"-z-0">>,1569421905590,ubf,postback,undefined,desktop,undefined,<<"1D07NHzFVMQhZicHBwct">>,{postback_event,<<"QA7WLFIMebYihKNMPvdSBpi13ouDaAVkAAl1bmRlZmluZWRkAAZ3aW5kb3drAAtzZWFyY2gtbGlzdGgEZAAGcmVuZGVyZAAGdXBkYXRlaARkAAZyZW5kZXJrAB9zZWFyY2gvc2VhcmNoLXF1ZXJ5LXdyYXBwZXIudHBsZAAJdW5kZWZpbmVkbAAAAANoAmQABnRhcmdldGsAC3NlYXJjaC1saXN0aAJkAAh0ZW1wbGF0ZWsAH3NlYXJjaC9zZWFyY2gtcXVlcnktd3JhcHBlci50cGxoAmQAB2NnX25hbWVrABVkZWZhdWx0X2NvbnRlbnRfZ3JvdXBqbAAAAANoAmQABnRhcmdldGsAC3NlYXJjaC1saXN0aAJkAAh0ZW1wbGF0ZWsAH3NlYXJjaC9zZWFyY2gtcXVlcnktd3JhcHBlci50cGxoAmQAB2NnX25hbWVrABVkZWZhdWx0X2NvbnRlbnRfZ3JvdXBqZAASYWN0aW9uX2Jhc2VfdXBkYXRl">>,<<"window">>,undefined,<<>>,{q,[{<<"qs">>,<<>>},{<<"type">>,<<"list">>},{<<"edge">>,[]},{<<"category.keyword">>,[<<"Object">>]},{<<"dc:type.keyword">>,[]},{<<"nave:collection.keyword">>,[]},{<<"nave:collectionPart.keyword">>,[]},{<<"date_start">>,[{<<"include_missing">>,false},{<<"max">>,<<"2018">>},{<<"min">>,<<"-4300">>}]},{<<"address_city.keyword">>,[]},{<<"dc:creator.keyword">>,[]},{<<"dc:subject.keyword">>,[]},{<<"dcterms:medium.keyword">>,[]},{<<"nave:technique.keyword">>,[]},{<<"facets">>,[{<<"nave:technique.keyword">>,[{<<"terms">>,[{<<"size">>,100},{<<"field">>,<<"nave:technique.keyword">>}]}]},{<<"dcterms:medium.keyword">>,[{<<"terms">>,[{<<"size">>,100},{<<"field">>,<<"dcterms:medium.keyword">>}]}]},{<<"dc:subject.keyword">>,[{<<"terms">>,[{<<"size">>,100},{<<"field">>,<<"dc:subject.keyword">>}]}]},{<<"dc:creator.keyword">>,[{<<"terms">>,[{<<"size">>,100},{<<"field">>,<<"dc:creator.keyword">>}]}]},{<<"address_city.keyword">>,[{<<"terms">>,[{<<"size">>,100},{<<"field">>,<<"address_city.keyword">>}]}]},{<<"date_start">>,[{<<"global">>,[{<<"aggs">>,[{<<"range_max">>,[{<<"max">>,[{<<"format">>,<<"YYYY">>},{<<"field">>,<<"date_start">>}]}]},{<<"range_min">>,[{<<"min">>,[{<<"format">>,<<"YYYY">>},{<<"field">>,<<"date_start">>}]}]}]}]}]},{<<"nave:collectionPart.keyword">>,[{<<"terms">>,[{<<"size">>,100},{<<"field">>,<<"nave:collectionPart.keyword">>}]}]},{<<"nave:collection.keyword">>,[{<<"terms">>,[{<<"size">>,100},{<<"field">>,<<"nave:collection.keyword">>}]}]},{<<"dc:type.keyword">>,[{<<"terms">>,[{<<"size">>,100},{<<"field">>,<<"dc:type.keyword">>}]}]},{<<"category.keyword">>,[{<<"terms">>,[{<<"size">>,100},{<<"field">>,<<"category.keyword">>}]}]}]}]}}}
16:31:45.806 [error] erfgoedbrabant z_transport:202 Stack: [
{erlydtl_runtime,to_value,[#{<<"_id">> => <<"6747f5f91816899e2ee9fd8c6c4f584cc4757d1a">>,<<"_index">> => <<"erfgoedbrabant_brabantcloud_a44a0d7d0ab0f91ec624a6ada9bc0a94978a944b">>,<<"_score">> => 0.0,<<"_source">> => #{<<"@id">> => <<"http://data.brabantcloud.nl/resource/aggregation/enb-test-objecten/enb.objecten-45bbd250-83bb-0adc-b458-69f3e5264633-edca0242-ca7d-dc66-7226-f7a536f9b59f">>,<<"@type">> => [#{<<"@id">> => <<"http://www.openarchives.org/ore/terms/Aggregation">>},#{<<"@id">> => <<"http://schemas.delving.eu/narthex/terms/Record">>}],<<"address_city">> => <<"Schijndel">>,<<"category">> => [<<"Object">>],<<"date_start">> => <<"1980-02-22">>,<<"dc:creator">> => [<<"Jan van Jansen">>,<<"Erfgoed Brabant">>],<<"dc:description">> => [<<"Gezicht op hoekhuis Harry Bolsiuslaan / Pompstraat richting Bunderstraat vanuit woonhuis Heesters. Achter het hoekhuis is een gedeelte van een schoolgebouw te zien. Links van de Pompstraat de tuin van het Sint Jozefklooster. In de verte voormalige boerderij aan de Bunderstraat. Paard en blauwgroene kar op de voorgrond. Wat figuren op de stoep. Ook de straten zijn wit van de sneeuw. Grijze lucht.">>],<<"dc:subject">> => [<<"Stadsgezicht">>,<<"Winter">>,<<"Onderwerp">>,<<"Straat">>],<<"dc:title">> => [<<"Titel wandkleden in Schijndel">>],<<"dc:type">> => [<<"Objectnaam Wandkleed">>],<<"edm:dataProvider">> => <<"Erfgoed Brabant">>,<<"http://www_europeana_eu/schemas/edm/aggregatedCHO">> => #{<<"@id">> => <<"http://data.brabantcloud.nl/resource/document/enb-test-objecten/enb.objecten-45bbd250-83bb-0adc-b458-69f3e5264633-edca0242-ca7d-dc66-7226-f7a536f9b59f">>,<<"@type">> => #{<<"@id">> => <<"http://www.europeana.eu/schemas/edm/ProvidedCHO">>},<<"http://purl.org/dc/elements/1.1/creator">> => [#{<<"@id">> => <<"http://data.brabantcloud.nl/resource/agent/enb-test-objecten/9abe8dcc-ddb2-db26-8e92-b23c89c541ca">>,<<"@type">> => #{<<"@id">> => <<"http://www.europeana.eu/schemas/edm/Agent">>},<<"http://rdvocab.info/ElementsGr2/biographicalInformation">> => #{<<"@value">> => <<"Hier staat niet de biografie.">>},<<"http://rdvocab.info/ElementsGr2/dateOfBirth">> => #{<<"@value">> => <<"1800-01-01">>},<<"http://rdvocab.info/ElementsGr2/dateOfDeath">> => #{<<"@value">> => <<"1840-09-25">>},<<"http://rdvocab.info/ElementsGr2/gender">> => #{<<"@value">> => <<"man">>},<<"http://rdvocab.info/ElementsGr2/placeOfBirth">> => #{<<"@id">> => <<"file:///mnt/brabantcloud-ingestion/hub3/brabantcloud/NarthexVersions/narthex-0.5.1/5884628">>},<<"http://rdvocab.info/ElementsGr2/placeOfDeath">> => #{<<"@id">> => <<"file:///mnt/brabantcloud-ingestion/hub3/brabantcloud/NarthexVersions/narthex-0.5.1/2744349">>},<<"http://rdvocab.info/ElementsGr2/professionOrOccupation">> => #{<<"@value">> => <<"Rol Tapijtwerker">>},<<"http://schemas.delving.eu/nave/terms/agentType">> => #{<<"@value">> => <<"persoon">>},<<"http://schemas.delving.eu/nave/terms/bibliographicalInformation">> => #{<<"@value">> => <<"Hier staat geen bibliografische data.">>},<<"http://schemas.delving.eu/nave/terms/creatorLabel">> => [#{<<"@value">> => <<"Hier staat normaliter de naam van de organisatie.">>},#{<<"@value">> => <<"Jan van Jansen">>}],<<"http://schemas.delving.eu/nave/terms/organisationLabel">> => #{<<"@value">> => <<"Hier staat normaliter de naam van de organisatie.">>},<<"http://schemas.delving.eu/nave/terms/periodOfLife">> => #{<<"@value">> => <<"19e eeuw">>},<<"http://www.w3.org/2004/02/skos/core#altLabel">> => [#{<<"@value">> => <<"Jantje">>},#{<<"@value">> => <<"Jen">>},#{<<"@value">> => <<"Jannus">>},#{<<"@value">> => <<"Jann">>},#{<<"@value">> => <<"Janus">>}],<<"http://www.w3.org/2004/02/skos/core#prefLabel">> => [#{<<"@value">> => <<"Jan van Jansen">>},#{<<"@value">> => <<"Hier staat normaliter de naam van de organisatie.">>}]},#{<<"@id">> => <<"http://data.brabantcloud.nl/resource/agent/enb-test-objecten/4d4f87c3-2412-89b7-59a1-7dc9c4951da7">>,<<"@type">> => #{<<"@id">> => <<"http://www.europeana.eu/schemas/edm/Agent">>},<<"http://rdvocab.info/ElementsGr2/professionOrOccupation">> => #{<<"@value">> => <<"Rol Uit...">>},...}],...},...},...}],...},...]```

**Comment by DorienD** [comment]
This is preventing us from releasing 0.51.1

**Comment by mworrell** [comment]
Looks like there is a letter `m` missing here.

**Comment by DorienD** [comment]
Ah! Awesom! Could this be released in a 0.51.2 version perhaps?

**Comment by mworrell** [comment]
@dorien, yes later today I will release 0.51.2

**Comment by mworrell** [comment]
Closed in #2194
